extensions/mentions/migrations/2022_05_20_000005_add_created_at_to_post_mentions_post_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('post_mentions_post', function (Blueprint $table) {
+            $table->timestamp('created_at')->nullable();
+        });
+
+        // do this manually because dbal doesn't recognize timestamp columns
+        $connection = $schema->getConnection();
+        $prefix = $connection->getTablePrefix();
+        $connection->statement("ALTER TABLE $prefix`post_mentions_post` MODIFY created_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP");
```

Comment:
As per tests failing, should this (and others) instead be....

```suggestion
        $connection->statement("ALTER TABLE `$prefixpost_mentions_post` MODIFY created_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP");
```

Reply:
yup, my bad

```suggestion
        $connection->statement("ALTER TABLE `{$prefix}post_mentions_post` MODIFY created_at TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP");
```

Reply:
I'll go through and update them -- no need to inconvenience Rafael with that :)

Reply:
Oh yeah, I don't have permission to push to GB's fork! 🙈 

The suggestions are all there for @iPurpl3x to bulk-commit anyway :)

Reply:
Done @davwheat